pyfar/dsp/dsp.py

```diff
@@ -2669,3 +2669,192 @@ def normalize(signal, reference_method='max', domain='auto',
         return normalized_signal, reference_norm
     else:
         return normalized_signal
+
+
+def correlate(signal_1, signal_2, mode='full', normalize=None):
```

Comment:
```suggestion
def correlate(signal_1, signal_2, mode='full', normalize=False):
```
Default is `False` according to the docstring